safe/definitions/post_processors.py

```diff
@@ -248,6 +313,8 @@ def post_processor_affected_function(**kwargs):
                 ],
             }]
     },
+    # output is described as ordered dict because the order is important
+    # and the postprocessor produce two fields.
```

Comment:
I think it's ok. We are producing rich reporting and we are not limited in the number of fields.